src/uct/api/uct.h

```diff
@@ -459,6 +462,17 @@ struct uct_iface_attr {
             } rndv;                      /**< Attributes related to rendezvous protocol */
         } tag;                           /**< Attributes for TAG operations */
 
+        struct {
+            size_t           max_iov;    /**< Maximal @a iovcnt parameter in
+                                              @ref uct_ep_mem_reg */
+            size_t           max_dim;    /**< Maximal dimension for a strided memory
+                                              layout with @ref uct_iov_t */
+            size_t           max_short;  /**< Maximal @a iovcnt parameter in
```

Comment:
inspired by "short" messages - up to some X iov-s UMR can post inline, and for >X latency is worse. the function will work either way, but UCP wants to act accordingly.